tb_device_mqtt.py

```diff
@@ -858,31 +859,26 @@ def _add_attr_request_callback(self, callback):
 
     def __timeout_check(self):
         while not self.stopped:
-            if not self.__timeout_queue.empty():
-                item = self.__timeout_queue.get_nowait()
-                if item is not None:
-                    while not self.stopped:
-                        current_ts_in_millis = int(time() * 1000)
-                        if current_ts_in_millis > item["ts"]:
-                            break
-                        sleep(0.2)
-
-                    with self._lock:
-                        callback = None
-                        if item.get("attribute_request_id"):
-                            if self._attr_request_dict.get(item["attribute_request_id"]):
-                                callback = self._attr_request_dict.pop(item["attribute_request_id"])
-                        elif item.get("rpc_request_id"):
-                            if self.__device_client_rpc_dict.get(item["rpc_request_id"]):
-                                callback = self.__device_client_rpc_dict.pop(item["rpc_request_id"])
-                    if callback is not None:
-                        if isinstance(callback, tuple):
-                            callback[0](None, TBTimeoutException("Timeout while waiting for a reply from ThingsBoard!"),
-                                        callback[1])
-                        else:
-                            callback(None, TBTimeoutException("Timeout while waiting for a reply from ThingsBoard!"))
-            else:
-                sleep(0.2)
+            current_ts_in_millis = int(time() * 1000)
```

Comment:
Looks like it will be better to use time.monotonic() instead of time.time(), to avoid affect of timezone changing.